### ISSUE TITLE
EN 15804+A2 chapter 7.3.2.1 Scenario information for Module A4, Transport to the building site

### ISSUE THREAD

**Issue by RobertSpang** [InDataWG/ILCD-EPD-Data-Format#11]
### I raise this change request because...

- [ ] the ILCD+EPD format lacks compliance with regualtions.
- [X] the ILCD+EPD format lacks conformaty with standards.
- [ ] the ILCD+EPD format needs to be improved regarding existing format or master data elements.
- [ ] there is another problem or use case I´m trying to address.

### Problem description

![EN 15804+A2 chapter 7 3 2 1 Scenario information for Module A4, Transport to the building site](https://github.com/InDataWG/ILCD-EPD-Data-Format/assets/150464594/de4038a0-08d2-46a3-b4f3-486bc8aa54e9)
The scenario information regarding Module A4 is currently not part of the format.

### Solution description

New master datasets need to be created and possibly the format extended

### Relevance or added value

Medium relevance

**Comment by RobertSpang** [comment]
Fuel type (has no unit) and fuel consumption (unit l/100 km) need two seperate lines, rest is well defined for implementation.

**Comment by okworx** [comment]
To be specified under "Technology description including background system" as free text.

Consensus is that this information will be subject to regulation in the future, and only once it is regulated and harmonized it will be possible to make it fully machine readable and usable.
